data/data.vcxproj

```diff
@@ -97,7 +97,7 @@
     <CustomBuildStep>
       <Message>Generating data</Message>
       <Outputs>difficulty1_patched</Outputs>
-      <Inputs>"$(SolutionDir)\bin\$(Platform)\$(Configuration)\extractor.exe"</Inputs>
+      <Inputs>$(SolutionDir)\bin\$(Platform)\$(Configuration)\extractor.exe</Inputs>
```

Comment:
I seem to remember this being one of the things that causes mutually exclusive breakage in different versions of MSVC (IE something like quotes break msvc2017, but no quotes breaks msvc2015). I guess from the VisualStudioVersion update above you tested this in the latest version? That's probably fine, as "get the latest version" is easier than "Get this specific old version that's hidden in MS' website somwehere"

Reply:
Indeed, msvc2017 complained about illegal characters in ``""path/to/extractor.exe""``

The 32bit build in Appveyor succeeded so it seems it has been fixed. However, the 64bit build exceeded the maximum build time (one hour) maybe there's something new going on? :thinking:

Reply:
Telling it to rebuild let is pass in the "Normal" time. *shrug*